source/includes/_quickstart-wallets.md

```diff
@@ -0,0 +1,42 @@
+## <em>Quickstart Wallets</em> 🚧🚧
+
+This guide gives a brief overview to how to do the following, using the wallet API:
+
+- Register parties
```

Comment:
Shouldn't be `Register companies`?

Reply:
In this case we ask them to provide the `owningPartyId` so I think it's clearer to say they give us a company and we respond with a party.